### PR TITLE
[major] Support non-interactive install & remove HPU mirror support

### PR DESCRIPTION
**Pull request by durera**
- Adds a non-interactive install mode
- There are no plans to support disconnected install for the existing HP Utilties application, removing the option to mirror images for this application.
- Removes redundant params: `mas_entitlement_username`, `mas_entitlement_key`, `sls_entitlement_username`, `sls_entitlement_key`
- Also updates and enhances documentation in various places.
- Also fixes various bugs around dependency version selection for the most recent two catalogs
- Fixes version of CP4D installed
- Fixes version of Db2 installed
- `echo_h2` function automatically calculates the list number

```
mas install -i mas1 -w ws1 -W "My Workspace" -c v8-amd64 --mas-channel 8.10.x --non-prod \
  --ibm-entitlement-key xxxxxxxxxxxx \
  --license-id xxxxxxxxxxxx --license-file /mnt/home/entitlement.lic \
  --uds-email xxxx.ibm.com --uds-firstname John --uds-lastname Barnes \
  --storage-rwo ibmc-block-gold --storage-rwx ibmc-file-gold-gid --storage-pipeline ibmc-file-gold-gid --storage-accessmode ReadWriteMany
```

### CODE REVIEW

File: image/cli/mascli/functions/mirror_to_registry [hunk 1]
```diff
@@ -550,24 +527,17 @@ function mirror_to_registry() {
...
     fi
-    # Assist, HPU & Predict Dependencies
-    if [[ "$MIRROR_MAS_ASSIST" == "true" || "$MIRROR_MAS_HPUTILITIES" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
+    # Assist & Predict Dependencies
+    if [[ "$MIRROR_MAS_ASSIST" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
       show_mirror_status " + IBM Cloud Pak for Data (CP4D) ...." $MIRROR_CP4D
     fi
-    # HPU & Predict Dependencies
-    if [[ "$MIRROR_MAS_HPUTILITIES" == "true" || "$MIRROR_MAS_PREDICT" == "true" ]]; then
-      show_mirror_status " + IBM Watson Studio ................" $MIRROR_WSL
```
Comment (whitfiea) [inline image/cli/mascli/functions/mirror_to_registry:559]: Do we not need WSL for Predict?

File: docs/commands/install.md [hunk 2]
```diff
@@ -48,36 +48,10 @@ A number of applications are not currently available when using a private mirror
 
 - Maximo Assist
 - Maximo Health & Predict Utilities
-- Maximo Monitor
 - Maximo Predict
-- Maximo Safety
 - Maximo Visual Inspection
 
 
-SNO Support
```
Comment (whitfiea) [inline docs/commands/install.md:57]: Why remove the SNO Support and Mustgather doc?

**Review by whitfiea — CHANGES_REQUESTED** [review]

File: docs/commands/install.md (hunk 2 above)
Comment (durera) [reply docs/commands/install.md:57]: - must-gather is out of date (replaced by dedicated new documentation https://ibm-mas.github.io/cli/commands/must-gather/
- sno section is a copy/paste of the airgap section plus a sentence that doesn't really add anything stating that we auto-detect SNO mode.

File: image/cli/mascli/functions/mirror_to_registry (hunk 1 above)
Comment (durera) [reply image/cli/mascli/functions/mirror_to_registry:559]: See line 540 in the changed file ... this now shows up in the Predict specific section

**Review by whitfiea — COMMENTED** [review]
@durera needs a changelog entry https://github.com/ibm-mas/cli/blob/master/docs/changes.md for 5.0.0

**Review by whitfiea — APPROVED** [review]
Approving, I am going to assume you have tested this non-interative install

**Comment by durera** [comment]
> Approving, I am going to assume you have tested this non-interative install

Yup 👍

**Pull request merged**